src/sources/youtube/PlayerEmbed.js

```diff
@@ -48,6 +48,7 @@ export default class YouTubePlayerEmbed extends React.Component {
         height="100%"
         autoplay
         modestBranding
+        disableKeyboard={true}
```

Comment:
to comply with a slightly pedantic lint rule:
```suggestion
        disableKeyboard
```